community/cypher/cypher-compiler-2.2_2.11/pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.neo4j</groupId>
+    <artifactId>cypher-parent</artifactId>
+    <version>2.2-SNAPSHOT</version>
```

Comment:
Looks like this needs an update, as the 2.2 branch is currently at version 2.2.1-SNAPSHOT. Same in line 13. (Thrilled to see Scala 2.11 support approaching btw!).